iam_assumerole/main.tf

```
@@ -45,6 +45,11 @@ variable "role_name" {
   description = "Name of the IAM role."
 }
 
+variable "permissions_boundary_policy_arn" {
+  description = "ARN of the permission boundary policy"
```

It should be clear that this MUST exist in order for this module to work. We should also indicate that this ARN must come from a policy that exists _outside_ of the module. Thus:
```suggestion
  description = <<EOM
(REQUIRED) ARN of an existing IAM policy (from another module/source)
which will be used as the Permissions Boundary for the IAM role.
CANNOT be left blank; policy MUST exist outside of this module!
EOM
```